GPT/gpt.py

```diff
@@ -65,6 +78,16 @@ def gpt_answer_question(text_to_process: str) -> str:
         """
         return gpt_query(prompt, text_to_process)
 
+    def model_blend(source_text: str, destination_text: str):
+        """Blend all the source text and send it to the destination"""
+
+        return model_blend(source_text, destination_text)
+
+    def model_blend_list(source_text: list[str], destination_text: str):
+        """Blend all the source text as a list and send it to the destination"""
+
+        return model_blend("\n---\n".join(source_text), destination_text)
```

Comment:
Is this working better than the original version which just used Python's built in list to string conversion?

Reply:
It's hard to say for certain, but I think so. It does give a little bit more context to the LLM to aid with how to handle multiple sources.